app/models/auth_token.rb

```diff
@@ -1,18 +1,60 @@
 # frozen_string_literal: true
 
-class AuthToken < ActiveRecord::Base
-  belongs_to :access_code
+class AuthToken < ActiveModelSerializers::Model
+  attributes :token, :expiration
 
-  validates :access_code, presence: true
-  validates :token, presence: true, uniqueness: true
-  validates :expiration, presence: true
+  class << self
+    EXPIRATION_CLAIM = "exp"
+    ISSUED_AT_CLAIM = "iat"
+    HMAC = "HS256"
 
-  before_validation { self.token = SecureRandom.uuid }
-  before_validation :set_expiration, on: :create
+    def generic_token
+      payload = {exp: new.expiration.to_i}
+      encode(payload)
+    end
 
-  private
+    def encode(payload)
+      payload = add_issued_at(payload)
```

Comment:
No that is the only thing in payload at this point. At this point it's basically a blank piece of paper that has mobile-content-api's notarized stamp, when the user tries to access one of our secure endpoints, they can include that piece of paper and we can make sure that it's our stamp and let the request through. Another analogy is a guest id-badge. No actual identifying info on the badge, but since we can verify that it's our badge and it's expiration date, that's all we need for now.

In #469 we add user_id to the payload, but I split these tickets up to demonstrate that the existing auth_code flow will still work.